sphinx/screens/dashboard/dashboard/src/main/res/values-zh/strings.xml

```diff
@@ -84,4 +84,7 @@
     <string name="dashboard_connect_retrieve_person_data_error">個人資訊下載失敗</string>
     <string name="dashboard_connect_message_empty">訊息不可無填</string>
     <string name="dashboard_connect_generic_error">聯絡人連結失敗，請稍後再試</string>
+
+    <string name="dashboard_network_disconnected_node_toast">断开的节点</string>
+    <string name="dashboard_network_connected_node_toast">连接节点</string>
```

Comment:
Use english translations until we get confirmed Chinese translations

Reply:
updated these in f1a318c